packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -78,6 +79,11 @@ class _DriverBinding extends BindingBase with SchedulerBinding, ServicesBinding,
 /// will still be returned in the `response` field of the result JSON along
 /// with an `isError` boolean.
 ///
+/// `enableTextEntryEmulation` will enable text entry emulation.
+/// Defaults to true. False can be used to enable manual testing
+/// with automation builds, otherwise text entry isn't working
+/// (see [FlutterDriver.setTextEntryEmulation]).
```

Comment:
Please start documentation as a grammatically correct English sentence with a capital letter. This documentation is also a bit unclear to someone not already familiar with the concept.

Maybe we could try

> The `enableTextEntryEmulation` parameter controls whether the application interacts with the system's text entry methods or a mocked out version used by Flutter Driver. If it is set to false, [FlutterDriver.enterText] will fail, but testing the application with real keyboard input is possible. This value may be updated during a test by calling [FlutterDriver.setTextEntryEmulation].

Reply:
Thanks! What you proposed sounds better indeed.